README.md

```diff
@@ -420,10 +420,11 @@ val sdJwtVcVerification = runBlocking {
         signer.issue(spec).getOrThrow()
     }
 
-    val verifier = SdJwtVcVerifier { did, _ ->
+    val verifier = SdJwtVcVerifier.usingDID { did, _ ->
```

Comment:
We need to change this example.
DID is not part of SD-JWT-VC

Reply:
Done.